docs-source/usersguide/application.rst

```diff
@@ -584,14 +584,27 @@ File                                 Parameters
 
 As a convenience, the file :file:`amber14-all.xml` can be used as a shortcut to include
 :file:`amber14/protein.ff14SB.xml`, :file:`amber14/DNA.OL15.xml`, :file:`amber14/RNA.OL3.xml`,
-and :file:`amber14/lipid17.xml`.  In most cases you can simply include that file,
-plus one of the water models.
+and :file:`amber14/lipid17.xml`.  In most cases, you can simply include that file,
+plus one of the water models, such as :file:`amber14/tip4pew.xml` for the TIP4P-Ew
+water model\ :cite:`Horn2004`:
+::
+
+    forcefield = ForceField('amber14-all.xml', 'amber14/tip4pew.xml')
+
+.. tip:: The solvent model XML files included under the :file:`amber14/` directory
+         include both water *and* ions compatible with that water model, so if you
+         mistakenly specify :file:`tip3p.xml` instead of :file:`amber14/tip3p.xml`,
+         you run the risk of neglecting critical parameters for ions in your system.
```

Comment:
I'm not sure that "run the risk of neglecting critical parameters" is quite the right way to put it.  They actual risk they run is having ForceField throw an exception because there aren't any residue templates for the ions.

Reply:
Good point---will correct.